gateway-ha/src/main/java/io/trino/gateway/ha/persistence/dao/QueryHistory.java

```diff
@@ -28,12 +28,12 @@
 public class QueryHistory
         extends Model
 {
-    private static final String queryId = "query_id";
-    private static final String queryText = "query_text";
-    private static final String backendUrl = "backend_url";
-    private static final String userName = "user_name";
-    private static final String source = "source";
-    private static final String created = "created";
+    public static final String queryId = "query_id";
```

Comment:
If they are public .. should they not be changed to QUERY_ID and such @ebyhr ?

Reply:
Thought so .. thanks for confirming

Reply:
looks like these fields have been removed in latest updates.